js/scripts.js

```diff
@@ -0,0 +1,19 @@
+var url = 'http://api.icndb.com/jokes/random';
+var button = document.getElementById('get-joke');
+var paragraph = document.getElementById('joke');
+
+button.addEventListener('click', function() {
+    getJoke();
+});
+
+function getJoke() {
+  var xhr = new XMLHttpRequest();
+  xhr.open('GET', url);
+  xhr.addEventListener('load', function() {
+    var response = JSON.parse(xhr.response);
+    paragraph.innerHTML = response.value.joke;
```

Comment:
the script will show undefined, or if the object gotten from the server has no response.value object at all it will show errors in console, because it can't get joke property from non-existing object :P